{% extends 'portal/base.html' %}

{% load static %}

{% block title %}
    Documentation
{% endblock %}

{% block content_header %}
 UniBind documentation
{% endblock %}
      
{% block breadcrumb %}
  <li><a href="/"><i class="fa fa-home"></i> Home</a></li>
  <li class="active">Docs</li>
{% endblock %}

{% block content %}

    <div class="row">
            
        <div class="col-md-8 col-lg-8 col-xs-12">

          <div class="box box-success">
            <div class="box-header with-border">
              <h3 class="box-title"><i class="fa fa-book"></i> UniBind Documentation</h3>
              <p class="pull-right"><em>Last updated: June 17, 2018</em></p>

              </div>
            <!-- /.box-header -->
            <div class="box-body">
              <h3 id="UniBind"> <i class="fa fa-question-circle"></i> What is UniBind?</h3>
              <hr>
              <b>UniBind</b> is a comprehensive map of direct transcription factor (TF) – DNA interactions in the human genome. These interactions were obtained by uniformly processing thousands of ChIP-seq data sets, from raw reads to high confidence TF binding site predictions, using the <a href="https://bitbucket.org/CBGR/chip-eat" target="_blank"> <b>ChIP-eat</b></a> software. ChIP-eat used the MACS2 peak caller to identify ChIP-seq peaks on the hg38 version of the human genome. Next, these genomic regions were analysed with four different TF binding models to predict direct TF-DNA interactions. These models include DiMO-optimized position weight matrices (PWMs), transcription factor flexible models, DNA shape-based models, and binding energy models. An entropy-based algorithm was used to automatically delineate an enrichment zone containing direct TF – DNA interactions, supported by both strong computational evidence and strong experimental evidence. The UniBind database hosts the complete set of TFBS predictions for each prediction model, as well as the models themselves, the original ChIP-seq peaks, and cis-regulatory modules derived from these direct TF – DNA interactions. All the data is publicly available. For further details, please refer to the associated publication: (DOI:….).
              
              <h3 id="dataprocess"><i class="fa fa-database"></i> How was the data processed?</h3>
              <hr>
              <p>
              The entire collection of ChIP-seq data sets was processed through the ChIP-eat data processing pipeline. The complete pipeline designed to process ChIP-seq data sets from raw reads to identification of direct TF-DNA binding events was implemented in the ChIP-eat software with source code freely available at <a href="https://bitbucket.org/CBGR/chip-eat/" target="_blank">https://bitbucket.org/CBGR/chip-eat/</a>. Specifically, ChIP-eat allows for:
              <ol>
                <li>aligning and filtering raw ChIP-seq data, </li>
                <li>calling ChIP-seq peaks, </li>
                <li>training the TFBS computational models and </li>
                <li>(iv)  automatically defining the enrichment zone in the landscape <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4082612/" target="_blank">plots</a> to predict TFBSs</li>
                <li>Four TFBS computation models are provided: PWMs, optimized with <a href="https://www.ncbi.nlm.nih.gov/pubmed/24369152" target="_blank">DiMO</a>, <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3764009/" target="_blank">TFFM</a>,
<a href="https://www.cell.com/cell-systems/abstract/S2405-4712(16)30218-6" target="_blank">DNAshaped</a>, 
and <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3389974/" target="_blank">binding energy models </a>).</li>

              </ol>
                 Only the ChIP-seq datasets for which a TF binding profile for the targeted TF was available in  <a href="http://jaspar2018.genereg.net/" target="_blank">JASPAR</a> were used for TFBS predictions. The enrichment zone containing high confidence direct TF-DNA interactions was automatically defined for each data set using an entropy-based algorithm. The diagram below illustrates the processing steps.
               </p>
              <img class="img-responsive img-rounded center-block" src="{% static 'img/chipeat.png' %}" width="90%">


              <h3 id="UniBindhost"><i class="fa fa-server"></i> What data does UniBind host?</h3>
              <hr>
              <p>The UniBind database contains millions of transcription factor (TF) binding site (TFBS) predictions across the human genome. These predictions were derived from the uniform processing of 1,983 publicly available ChIP-seq datasets accounting for 232 distinct TFs in 346 cell lines. For each ChIP-seq data set, the user can download the following: the set of predicted direct TF-DNA interactions in  <a href="https://genome.ucsc.edu/FAQ/FAQformat.html" target="_blank">BED6</a> format, and in <a href="https://en.wikipedia.org/wiki/FASTA_format" target="_blank">FASTA </a> format, the initial set of ChIP-seq peaks, a visual representation of the delineated enrichment zone, and the trained prediction model used. For each ChIP-seq data set present in UniBind, four prediction models were used (see previous section) and the data is available for download for each of the four models individually.</p>

              <h3 id="UniBindentry"><i class="fa fa-file"></i> What does each entry in UniBind contain?</h3>
              <hr>
              <p>WWhen searching/clicking on one TF name or one ChIP-seq data set of interest, a page will be displayed with all the information available. A summary part is at the top of the page describing what cell line, TF, data source, and JASPAR profile was used in the prediction models. The user is also provided with external links pointing for more details about this entry’s components.</p>

              <p>On the middle section of the page, all the prediction models used for this database entry are displayed in a tabbed layout. Download buttons are also available in this section from where the user can obtain the set of TF binding site predictions and/or the original set of ChIP-seq peaks. At the bottom of this section, a plot consisting of four panels is displayed to give a visual representation of the data through the selected prediction model. Please see next section in documentation on how to interpret these plots.</p>

              <p>Below this section, a summary of the statistics for all the prediction models used for this data set is displayed. Namely, for each prediction model, the thresholds that define the set of direct TF-DNA interactions are displayed, as well as a  <a href="http://meme-suite.org/doc/centrimo.html" target="_blank">CentriMo</a>  p-value providing information about the centrality of the predictions with respect to the ChIP-seq peak summits. The lower the p-value the stronger the centrality. The closer it is to 0 the lower the quality of the data and/or performance of the prediction model is on this particular data set.</p>
              <p>
              Finally, at the bottom of the entry page, suggestions with other ChIP-seq data sets targeting the same TF are available.</p>

              <h3 id="download"><i class="fa fa-download"></i> How can I download the data?</h3>
              <hr>
              <p>The user can download the set of transcription factor binding sites (TFBSs), the initial ChIP-seq peaks, and the trained prediction models for each of the datasets individually by navigating to the UniBind entry of interest. If the user wants to download the available data in bulk, this can be done through the <a href="/downloads">Download</a> section of the UniBind database. This section allows the user to download the cis-regulatory modules derived using the entire set of TFBS predictions.</p>


              <h3 id="centrality"><i class="fa fa-align-center"></i> How to interpret the enrichment zone and centrality p-value?</h3>
              <hr>
              <p> 
              <img class="img-responsive img-rounded center-block" src="{% static 'img/example_plot.png' %}" width="90%">

            For each entry in <b>UniBind</b>, a visual representation of the results is available. It consists of a plot composed of four panels. Each point in panel (A) represents the top scoring sequence in one ChIP-seq peak, that is the sequence within the ChIP-seq peak with the best score computed from the TFBS computation model used. On the Y-axis is the motif score, and on the X-axis is the distance to the ChIP-seq peak summit. The dashed lines represent the thresholds on the TFBS computational model scores and on the distance to the peak summits, delimiting the area that contains direct TF-DNA interactions. Panel (B) is a heat map of the data from panel (A) to help in visualizing the density of the data and better understand the defined thresholds. The bottom panels show where the threshold on the motif score (C) and on the distance from the peak summit (D) was automatically defined by the entropy-based algorithm. In the legend of panel (D) the centrality p-value is provided.  The closer it is to 0 the lower the quality of the data and/or performance of the prediction model is on this particular data set.</p>

              <h3 id="search"><i class="fa fa-search"></i> How to use advanced search?</h3>
              <hr>
              <p>The user can perform an advanced search of the data available in <b>UniBind</b>. Entering the Search page by clicking “Search” on the top of the page, an “Advanced Options” link is available under the search bar. Here, the user can select the TF of interest, the cell-line of interest, as well as the preferred prediction model, or data source. Another option is to browse only the high quality data sets by checking the option P-value. This will filter based on the data sets that present a significant centrality p-value (log10(p-value) < 0). Once the desired options are selected, the user can click the search button, and come back to modify those options at any time during the browsing. The diagram below illustrates the use of advanced search:</p>

              <img class="img-responsive img-rounded center-block" src="{% static 'img/UniBind.png' %}" width="90%">

              <hr>
              <b>ChIP-eat pipeline source code is available on <a href="https://bitbucket.org/CBGR/chip-eat">Bitbucket</a></b>

              <!-- code folding -->
            </div>
            <!-- /.box-body -->
          </div>
          <!-- /.box -->
        </div>

      <div class="col-md-4 col-lg-4 col-xs-12">
          <div class="box box-success">
            <div class="box-header with-border">
              <h3 class="box-title"><i class="fa fa-list"></i> Table of contents</h3>

              </div>
            <!-- /.box-header -->
            <div class="box-body">
            <div id="TOC">
              <ul>
      
              <li><a href="#UniBind">How is UniBind?</a></li>
              <li><a href="#dataprocess">How was the data processed?</a></li>
              <li><a href="#UniBindhost">What data does UniBind host?</a></li>
              <li><a href="#UniBindentry">What does each entry in UniBind contain?</a></li>
              <li><a href="#download">How can I download the data?</a></li>
              <li><a href="#centrality">How to interpret the enrichment zone and centrality p-value?</a></li>
              <li><a href="#search">How to use advanced search?</a></li>
              <li><a href="/contact-us">Contact</a></li>
              </ul>
              </div>
            </div>
            <!-- /.box-body -->
          </div>
          <!-- /.box -->
        </div>
        
        
  </div>

    {% block javascript %}
      <script>

              // add bootstrap table styles to pandoc tables
              $(document).ready(function () {
                $('tr.header').parent('thead').parent('table').addClass('table table-condensed');
              });

              </script>

              <!-- dynamically load mathjax for compatibility with self-contained -->
              <script>
                (function () {
                  var script = document.createElement("script");
                  script.type = "text/javascript";
                  script.src  = "https://cdn.mathjax.org/mathjax/latest/MathJax.js?config=TeX-AMS-MML_HTMLorMML";
                  document.getElementsByTagName("head")[0].appendChild(script);
                })();
              </script>

        <script>
        $(document).ready(function () {
          window.buildTabsets("TOC");
        });
        </script>

    {% endblock %}

  

{% endblock %}