{% extends 'portal/base.html' %}

{% load static %}

{% block title %}
    Documentation
{% endblock %}

{% block content_header %}
 UniBind documentation
{% endblock %}
      
{% block breadcrumb %}
  <li><a href="/"><i class="fa fa-home"></i> Home</a></li>
  <li class="active">Docs</li>
{% endblock %}

{% block content %}

    <div class="row">
            
        <div class="col-md-8 col-lg-8 col-xs-12">

          <div class="box box-success">
            <div class="box-header with-border">
              <h3 class="box-title"><i class="fa fa-book"></i> UniBind Documentation</h3>
              <p class="pull-right"><em>Last updated: June 17, 2018</em></p>

              </div>
            <!-- /.box-header -->
            <div class="box-body">
              <h3 id="UniBind"> <i class="fa fa-question"></i> What is UniBind?</h3>
              <hr>
              <b>UniBind</b> is a comprehensive map of direct transcription factor (TF) – DNA interactions in the human genome. These interactions were derived by uniformly processing thousands of ChIP-seq data sets from the raw reads to high confidence TF binding site predictions. The <a href="https://bitbucket.org/CBGR/chip-eat" target="_blank"> <b>ChIP-eat</b></a> data processing pipeline was specifically developed and used in this scope, which is illustrated in figure below. After aligning the raw reads to the hg38 version of the human genome, the MACS2 peak caller was used to identify genomic regions of interest, and four different prediction models, varying from simple to complex, were used to derive TF binding sites. These models include DiMO-optimized position weight matrices, transcription factor flexible models, DNA shape features, and binding energy models. An entropy-based algorithm was used on each set of predictions, to automatically delineate an enrichment zone containing direct TF – DNA interactions, supported by both strong computational evidence and strong experimental evidence. The UniBind database hosts the complete set of TFBS predictions from each prediction model, as well as the models themselves, the original ChIP-seq peaks, and the cis-regulatory modules derived from the direct TF – DNA interactions. All the data is publicly available. For further details, please refer to the associated publication: (DOI: ).
              
              <h3 id="dataprocess"><i class="fa fa-data"></i> How was the data processed?</h3>
              <hr>
              <p>
              The entire collection of ChIP-seq datasets was processed through the ChIP-eat data processing pipeline. ChIP-eat was specifically developed for the uniform processing of ChIP-seq data. This pipeline is publicly available at <a href="https://bitbucket.org/CBGR/chip-eat/" target="_blank">https://bitbucket.org/CBGR/chip-eat/</a> for download and implementation, and it consists of three main parts:
              <ol>
                <li>Raw read filtering and alignment (<a href="https://en.wikipedia.org/wiki/FASTA_format" target="_blank">FASTA</a> format) to the hg38 version of the human genome, </li>
                <li>ChIP-seq peak calling using <a href="www.ncbi.nlm.nih.gov/pmc/articles/PMC2592715/" target="_blank">MACS2</a>, and</li>
                <li>the processing of the ChIP-seq peaks using one or all four supported TFBS prediction models (<a href="https://www.ncbi.nlm.nih.gov/pubmed/24369152" target="_blank">DiMO</a> optimized PWMs, <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3764009/" target="_blank">TFFM</a>,
<a href="https://www.cell.com/cell-systems/abstract/S2405-4712(16)30218-6" target="_blank">DNAshaped</a>, 
and <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3389974/" target="_blank">BEM</a>). These prediction models vary from simple to complex, as it has been shown that there is no “one-fits-all” model for TFBS prediction.</li>

              </ol>
                 Only the ChIP-seq datasets for which the targeted transcription factor contained a binding profile in the <a href="http://jaspar2018.genereg.net/" target="_blank">JASPAR</a> database were used in the third processing step. The enrichment zone containing high confidence TF-DNA interactions was automatically defined for each dataset using an entropy-based algorithm. The diagram below illustrates the processing steps.

                 The quality of the dataset and of the predictions is reflected by the <a href="http://meme-suite.org/doc/centrimo.html" target="_blank">CentriMo</a> - like logged p-value.

               </p>
              <img class="img-responsive img-rounded center-block" src="{% static 'img/chipeat.png' %}" width="90%">

              <h3 id="UniBindhost"><i class="fa fa-search"></i> What data does UniBind host?</h3>
              <hr>
              <p>The UniBind database contains millions of transcription factor (TF) binding site (TFBS) predictions across the human genome. These predictions were derived from the uniform processing of 1,983 publicly available ChIP-seq datasets accounting for 232 distinct TFs in 346 cell lines. For each ChIP-seq dataset, the user can download the following: the set of predicted direct TF-DNA interactions, in <a href="https://genome.ucsc.edu/FAQ/FAQformat.html" target="_blank">BED6</a> format, and in <a href="https://en.wikipedia.org/wiki/FASTA_format" target="_blank">FASTA </a> format, the initial set of ChIP-seq peaks, a visual representation of the delineated enrichment zone, and the trained prediction model used. For each ChIP-seq dataset present in UniBind, four prediction models were used (see previous section) and the data is available for download for each of the four models individually.</p>

              <h3 id="UniBindentry"><i class="fa fa-search"></i> What does each entry in UniBind contain?</h3>
              <hr>
              <p>When searching/clicking on one transcription factor (TF) or one ChIP-seq dataset of interest, a page will be displayed with all the information available. A summary part is at the top of the page describing what cell line, TF, data source and <a href="http://jaspar2018.genereg.net/" target="_blank">JASPAR</a> profile was used in the prediction models. The user is also provided with external links pointing for more details about this entry’s components.</p>
              <p>On the middle section of the page, all the prediction models used for this database entry are displayed in a tabbed layout. Download buttons are also available in this section from where the user can obtain the set of TF binding site predictions and/or the original set of ChIP-seq peaks. At the bottom of this section, a plot consisting of four panels is displayed to give a visual representation of the data through the selected prediction model. Please see next section on how to interpret these plots.</p>
              <p>Below this section, a summary of the statistics for all the prediction models used for this dataset is displayed. Namely, for each prediction model, the thresholds that define the set of direct TF-DNA interactions are displayed, as well as a <a href="http://meme-suite.org/doc/centrimo.html" target="_blank">CentriMo</a>-like p-value. This centrality logged p-value is an indication of the quality of the data and of the performance of the prediction model. The lower the p-value the better the quality. A centrality p-value equal to 0 is considered insignificant, and denotes poor data quality or poor performance of the prediction model on this particular dataset. In general , the closer it is to 0 the lower the quality of the data and/or performance of the prediction model is.</p>
              <p>
              Finally, at the bottom of the entry page, suggestions with other ChIP-seq datasets targeting the same TF are made where available.</p>

              <h3 id="download"><i class="fa fa-download"></i> How can I download the data?</h3>
              <hr>
              <p>The user can download the set of transcription factor binding sites (TFBSs), the initial ChIP-seq peaks, and the trained prediction models for each of the datasets individually by navigating to the UniBind entry of interest. If the user wants to download the available data in bulk, this can be done through the <a href="/downloads">Download</a> section of the UniBind database. This section allows the user to download the cis-regulatory modules derived using the entire set of TFBS predictions.</p>



              <h3 id="centrality"><i class="fa fa-search"></i> How to interpret the enrichment zone and centrality p-value?</h3>
              <hr>
              <p> 
              <img class="img-responsive img-rounded center-block" src="{% static 'img/example_plot.png' %}" width="90%">

            For each entry in UniBind and for each prediction model used, a visual representation of the data is available. It consists of a plot composed of four panels. Each point in panel (A) represents the top scoring sequence in one ChIP-seq peak, that is the sequence within the ChIP-seq peak that was the closest to the <a href="http://jaspar2018.genereg.net/" target="_blank">JASPAR</a> profile. On the Y-axis is the motif score assigned by the prediction model, and on the X-axis is the distance of the top scoring sequence in the ChIP-seq peak from the peak summit (i.e., where the most reads aligned to). The dashed lines represent the thresholds on the motif scores and on the distance to the peak summits, delimiting the area that contains direct TF-DNA interactions. Panel (B) is a heat map of the data from panel (A) to help in visualizing the density of the data and better understand the defined thresholds. The bottom panels show where the threshold on the motif score (C ) and on the distance from the peak summit (D) was automatically defined by the entropy-based algorithm. Each of the two plots contains the histogram of the motif scores and of the distances to the peak summit. The dashed line represents the actual threshold set. In the legend of panel (D) the <a href="http://meme-suite.org/doc/centrimo.html" target="_blank">CentriMo</a>-like logged p-value indicates the quality of the data and/or the performance of the prediction model. A centrality p-value that is equal to 0 is considered insignificant. If insignificant or closer to 0, it denotes poor data quality and/or low performance of the prediction model on this particular dataset. If the centrality p-values is “NA” it means that either one, or both thresholds were not able to be set due to the poor quality of the data.</p>

              <h3 id="search"><i class="fa fa-search"></i> How to use advanced search?</h3>
              <hr>
              <p>The user can perform an advanced search of the data available in UniBind. Entering the Search page by clicking “Search” on the top of the page, an “Advanced Options” link is available under the search bar. Here, the user can select the transcription factor (TF) of interest, the cell-line of interest, as well as the preferred prediction model or data source. Another option is to browse only the high quality datasets by checking the option P-value. This will filter out the datasets that present an insignificant centrality p-value (see this (link to “What does each entry in UniBind contain ?”) section). Once the desired options are selected, the user can click the search button, and come back to modify those options at any time during the browsing. The diagram below illustrates the use of advanced search:</p>

              <img class="img-responsive img-rounded center-block" src="{% static 'img/UniBind.png' %}" width="90%">

              <hr>
              <b>ChIP-eat pipeline source code is available on <a href="https://bitbucket.org/CBGR/chip-eat">Bitbucket</a></b>

              <!-- code folding -->
            </div>
            <!-- /.box-body -->
          </div>
          <!-- /.box -->
        </div>

      <div class="col-md-4 col-lg-4 col-xs-6">
          <div class="box box-success">
            <div class="box-header with-border">
              <h3 class="box-title"><i class="fa fa-list"></i> Table of contents</h3>

              </div>
            <!-- /.box-header -->
            <div class="box-body">
            <div id="TOC">
              <ul>
      
              <li><a href="#UniBind">How is UniBind?</a></li>
              <li><a href="#dataprocess">How was the data processed?</a></li>
              <li><a href="#UniBindhost">What data does UniBind host?</a></li>
              <li><a href="#UniBindentry">What does each entry in UniBind contain?</a></li>
              <li><a href="#download">How can I download the data?</a></li>
              <li><a href="#centrality">How to interpret the enrichment zone and centrality p-value?</a></li>
              <li><a href="#search">How to use advanced search?</a></li>
              <li><a href="/contact-us">Contact</a></li>
              </ul>
              </div>
            </div>
            <!-- /.box-body -->
          </div>
          <!-- /.box -->
        </div>
        
        
  </div>

    {% block javascript %}
      <script>

              // add bootstrap table styles to pandoc tables
              $(document).ready(function () {
                $('tr.header').parent('thead').parent('table').addClass('table table-condensed');
              });

              </script>

              <!-- dynamically load mathjax for compatibility with self-contained -->
              <script>
                (function () {
                  var script = document.createElement("script");
                  script.type = "text/javascript";
                  script.src  = "https://cdn.mathjax.org/mathjax/latest/MathJax.js?config=TeX-AMS-MML_HTMLorMML";
                  document.getElementsByTagName("head")[0].appendChild(script);
                })();
              </script>

        <script>
        $(document).ready(function () {
          window.buildTabsets("TOC");
        });
        </script>

    {% endblock %}

  

{% endblock %}